content/rs/references/cli-utilities/rladmin/info.md

```diff
@@ -0,0 +1,153 @@
+---
+Title: rladmin info
+linkTitle: info
+description: Lists configurable settings for specified databases, proxies, clusters, or nodes.
+weight: $weight
+alwaysopen: false
+toc: "true"
+headerRange: "[1-2]"
+categories: ["RS"]
+aliases:
+---
+
+`rladmin info` lists configurable settings for specified databases, proxies, clusters, or nodes.
+
+## `info db`
+
+`rladmin info db` lists configurable settings for all databases.
+
+`rladmin info db <db:id | name>` lists configurable settings for the specified database.
+
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| db:id     | database id of the specified database |
+| name      | database name of the specified database |
+
+### Example
+
+``` sh
+$ rladmin info db db:1
+db:1 [database1]:
+    client_buffer_limits: 1GB (hard limit)/512MB (soft limit) in 30 seconds
+    slave_buffer: auto
+    pubsub_buffer_limits: 32MB (hard limit)/8MB (soft limit) in 60 seconds
+    proxy_client_buffer_limits: 0KB (hard limit)/0KB (soft limit) in 0 seconds
+    proxy_slave_buffer_limits: 1GB (hard limit)/512MB (soft limit) in 60 seconds
+    proxy_pubsub_buffer_limits: 32MB (hard limit)/8MB (soft limit) in 60 seconds
+    repl_backlog: 1.02MB (1073741 bytes)
+    repl_timeout: 360 seconds
+    repl_diskless: default
+    master_persistence: disabled
+    maxclients: 10000
+    conns: 5
+    conns_type: per-thread
+    sched_policy: cmp
+    max_aof_file_size: 300GB
+    max_aof_load_time: 3600 seconds
+    dedicated_replicaof_threads: 5
+    max_client_pipeline: 200
+    max_shard_pipeline: 2000
+    max_connections: 0
+    oss_cluster: disabled
+    oss_cluster_api_preferred_ip_type: internal
+    gradual_src_mode: disabled
+    gradual_src_max_sources: 1
+    gradual_sync_mode: auto
+    gradual_sync_max_shards_per_source: 1
+    slave_ha: disabled (database)
+    mkms: enabled
+    oss_sharding: disabled
+    mtls_allow_weak_hashing: disabled
+    mtls_allow_outdated_certs: disabled
+    data_internode_encryption: disabled
+    proxy_policy: single
+    db_conns_auditing: disabled
+```
+
+## `info proxy`
+
+`rladmin info proxy` lists configurable settings for the proxy.
+
```

Comment:
Missing `rladmin info proxy { <id> | all }` syntax and the parameters table.

Reply:
Fixed in [1f2429e](https://github.com/RedisLabs/redislabs-docs/pull/1887/commits/1f2429ea1f5084c471bd704a33f206640e2813ec)